src/wp-includes/option.php

```diff
@@ -643,50 +643,150 @@ function wp_load_alloptions( $force_cache = false ) {
 }
 
 /**
- * Loads and primes caches of certain often requested network options if is_multisite().
+ * Primes specific network options for the current network into the cache with a single database query.
  *
- * @since 3.0.0
- * @since 6.3.0 Also prime caches for network options when persistent object cache is enabled.
+ * Only network options that do not already exist in cache will be loaded.
+ *
+ * If site is not multisite, then call wp_prime_option_caches().
+ *
+ * @since 6.6.0
+ *
+ * @see wp_prime_network_option_caches()
+ *
+ * @param string[] $options An array of option names to be loaded.
+ */
+function wp_prime_site_option_caches( array $options = array() ) {
+	wp_prime_network_option_caches( null, $options );
```

Comment:
The updates in https://github.com/WordPress/wordpress-develop/commit/482cc816ea9f970611f8058b2a071cefc3cfe337 are a nice change, but I would also update the call to `wp_prime_network_option_caches()` here so it doesn't pass a `null` value.

```suggestion
	wp_prime_network_option_caches( 0, $options );
```

Reply:
@joemcgill @mukeshpanchal27 Passing null is what is done in functions like `get_network_option`. Why pass null there and 0 here?

Reply:
Interesting. I'd argue that `get_site_option()` (which is the only place where I see `null` being passed to `get_network_option()`) should also be passing `0`, but I think the consistency between the two sets of functions makes sense.